### ISSUE TITLE
Implementation of "Previous Lesson" button 

### ISSUE THREAD

**Issue by FrancisJTR7** [TheOdinProject/theodinproject#2623]
--#### Description
--I believe there should be an implementation of a "Previous Lesson"  button on the lesson button group. As I have gone through this curriculum many times have I needed to go back and revisit previous material but to do so I've had to go through "View Course" and manually find where I needed to go. This to me is a problem because sometimes I'll just need to go back to one lesson. Although it's not the biggest issue in the world, I believe it would offer some a better user flow while bringing no negative effects.  

--#### Acceptance Criteria:

--This feature would allow the user to go back to a lesson by simply clicking one button as opposed to browsing through the whole curriculum to do so instead. 

1. Log in

2. Visit a lesson page

3. Click the previous lesson button

4. The course will go back to the previous lesson

**Comment by CouchofTomato** [comment]
@KevinMulhern 

Wasn't this discussed recently elsewhere? Was there a consensus on whether we would bring back the button?

**Comment by 01zulfi** [comment]
I'm fully onboard with adding a previous lesson button

**Comment by KevinMulhern** [comment]
Yeah, we'll be bringing it back. But we still need an easy and accessible way to return to the course. We're going to implement that as part of the refresh. Leave it with me 😄

**Comment by Memomer** [comment]
agree with @jonro2955 and his idea for adding a menu, I have one more idea to optimize the process incase we don't go for the previous button idea which can be implemented easily.  So I will re-iterate the issue before I put forth the solution,  incase a user wants to go to the previous lesson he has to click on 'view course' button at the bottom of the page to check the course list then scroll down and check the desired lesson tab. Can we just add an functionality to view course button so that it auto-scrolls to that particular sub-section which reduce the scroll time. For example, I am on  a lesson from Git-basics sub section, so when i click on view course page auto-scrolls to Git-basics sub section.

Recommending this idea as it is a easy/lazy fix of sorts. Ofcourse, I did love to contribute if we go ahead with this idea.

**Comment by goldj12s** [comment]
It was mentioned in #1639 and #1002, which in both threads looked like it was not a popular option. However, it has been 5 years so this issue should be revisited.

**Comment by jai-sanghvi** [comment]
I'd like to suggest adding keyboard shortcuts for navigating the lessons as well. Maybe `Ctrl` + `←` to navigate to the previous lesson and `Ctrl` + `→` to navigate to the next lesson. For reference, this functionality is available for the lessons on [javascript.info](https://javascript.info/strict-mode)

**Comment by Ellin** [comment]
Just wanted to add that I would also really appreciate a "Previous Lesson" button. Right now, there are three buttons at the bottom of each lesson with "Next Lesson" being on the rightmost side. I think there should be a corresponding "Previous Lesson" button on the leftmost side, where the "View Course" button currently is. It breaks my flow to not be able to go back and forth between lessons — instead, I find myself _very frequently_ having to first click "View Course" and manually find the previous lesson. This has quickly gotten quite annoying.

I don't think the "View Course" button should be removed entirely, but imo it would make for a more natural user experience to have "Previous Lesson" where that button is instead. I certainly find myself needing to go back to the immediately preceding lesson while learning and doing projects much more often than wanting to "View Course".

**Comment by juliabush** [comment]
Hey I would like to work on this if possible I feel like this would be a great fix and is really needed. In one of the comments @KevinMulhern you mention "site roadmap" so not sure if this is open to being taken on by an assignee.

**Comment by KevinMulhern** [comment]
Thanks for dropping your thoughts and interest @Ellin and @juliabush!

There are a few prerequisites I'd like to have in place before doing this - chiefly still having an easy way back to the course. I have some rough ideas about how to do that. Leave that with me for a couple of weeks
